src/components/Card/index.examples.js

```diff
@@ -34,7 +35,7 @@ storiesOf('Cards')
 
   .addWithInfo('Lesson Card', () => (
     <div className={representClasses}>
-      <LessonCard response={response} />
+      <LessonCard response={lessonResponse} />
```

Comment:
You could actually use `<Request url={lessonUrl} />` with a headers wrapper like I do in the Instructor Center (https://github.com/eggheadio/egghead-instructor-center/blob/master/src/components/Request/index.js) if you want to hit the real endpoint so you don't have to update example data whenever the endpoint structure changes; but not a big deal right now - could be a feature later on.